src/send/SendAmount/useTransactionCallbacks.ts

```diff
@@ -78,7 +78,7 @@ function useTransactionCallbacks({
       localCurrencyExchangeRate,
       localCurrency: localCurrencyCode,
       localCurrencyAmount: localAmount?.toString() ?? null,
-      underlyingTokenAddress: transferTokenAddress,
+      underlyingTokenAddress: tokenInfo!.address!,
```

Comment:
same here...

Reply:
Fixed in https://github.com/valora-inc/wallet/commit/83caddecdff9f0e497f29222f60eb4fb2a1c5fd8!